### PR TITLE
Target .NET 3.5 for console and test projects

### PR DESCRIPTION
**Pull request by bobbertoriley**
Original blurb from the Discord:

```
Microsoft released Visual Studio 2022 about a month ago. I've started playing around with it since it's something
work wants us to start using. The main takeaway w.r.t. Sentinels mod development is that you can no longer install
versions of the .net framework/sdk <= 4.5.2 other than 3.5. (Also, by default, it only installs .net 6.0)

This is fine for the library itself, but using the workshop mod as an example, the Console + Testing libraries do not work 
out of the box because they target 4.0. I believe targeting 3.5 or 4.8 for both projects works, but I haven't done a bunch
of testing on either (I jumped my personal projects to 6-- apart from the json serialization needing rewritten for
loading/saving games and user preferences, everything else works!). Both of which have LTS up to at least 2028, so
you should be safe to pin the non-library projects to those.

tl;dr - If people don't read the instructions and see 'oh, I need to install Visual Studio' and install 2022, they may
have some hurdles getting the projects working initially.
```

I went with 3.5 so it aligns with the rest of the project; Visual Studio 2022 includes 4.8 if you install the .net build environment, so that would be easy enough to target higher instead.

### CODE REVIEW

File: MyModTest/BaseTest.cs [hunk 1]
```diff
@@ -4914,7 +4914,7 @@ protected GameController LoadGame(string name, bool addDataPath = false, bool ad
                 {
                     var dllpath = System.Reflection.Assembly.GetExecutingAssembly().CodeBase;
                     path = Path.GetDirectoryName(dllpath.Replace("file://", "")).Replace("\\D:", "D:").Replace("\\C:", "C:");
-                    path = Path.Combine(path, "..", "..", "DataFiles", name);
+                    path = new string[] { path, "..", "..", "DataFiles", name }.Aggregate((x, y) => Path.Combine(x, y));
```
Comment (bobbertoriley) [inline MyModTest/BaseTest.cs:4917]: This was the only breaking change between 3.5 and 4.x. `Path.Combine` only supported multiple params in 4.0+
